blkmaker.c

```diff
@@ -459,6 +459,7 @@ static const unsigned char witness_magic[] = { 0x6a /* OP_RETURN */, 0x24, 0xaa,
 #define commitment_spk_size (sizeof(witness_magic) + sizeof(libblkmaker_hash_t) /* witness mrklroot */)
 #define commitment_txout_size (8 /* value */ + 1 /* scriptPubKey length */ + commitment_spk_size)
 static const size_t max_witness_commitment_insert = max_varint_size + commitment_txout_size - 1;
+static const libblkmaker_hash_t witness_nonce = {0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
{} or {0} would be fine, didn't know what color to paint the shed :)

I just wanted it as explicit as possible that it was a value of exactly zero, as opposed to null as a default initialization value.